pkg/operators/v1/olmconfig_types.go

```diff
@@ -0,0 +1,58 @@
+package v1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// OLMConfigSpec is the spec for an OLMConfig resource.
+type OLMConfigSpec struct {
+	Features Features `json:"features,omitempty"`
+}
+
+// Features contains the list of configurable OLM features.
+type Features struct {
+	// +kubebuilder:default=false
+
+	// DisableCopiedCSVs is used to disable OLM's "Copied CSV" feature
+	// for operators installed at the cluster scope, where a cluster
+	// scoped operator is one that has been installed in an
+	// OperatorGroup that targets all namespaces.
+	// When reenabled, OLM will recreate the "Copied CSVs" for each
+	// cluster scoped operator.
+	DisableCopiedCSVs bool `json:"disableCopiedCSVs,omitempty"`
```

Comment:
You may need to make this a pointer to a bool to avoid `disableCopiedCSVs: false` being implicitly specified (or printed when firing off `kubectl get olmconfigs <...> -o yaml`).

Reply:
Ha! Actually landed on this change while consuming the API. Done!

Reply:
This was marked as resolved, but noticed that its not (or no longer) a pointer to a bool?